### ISSUE TITLE
adress bar clears when rotating to landscape mode

### ISSUE THREAD

**Issue by Meteor0id** [thunderbird/thunderbird-android#3049]
### Expected behavior
Rotating should not affect any of the typed content, not the body nor the address fileds or subject field

### Actual behavior
When typing an address and rotating while the field is still selected, the field is cleared. Rotating back does not make the text reappear.

### Steps to reproduce
1. start typing in the to: field
2. rotate to landscape mode

### Environment
K-9 Mail version:
5.401

Android version:
7.0 (on a moto G5 plus)

Account type (IMAP, POP3, WebDAV/Exchange):
IMAP

**Comment by philipwhiuk** [comment]
On the build on my phone we seem to keep complete tokens but discard addresses still being typed

e.g. for

> `a@b.com` f 

on rotate you are left with

> `a@b.com`

**Comment by aucklander** [comment]
you gotta complete the email address before rotating otherwise it will clear which it shouldn't do.

**Comment by dabbler011** [comment]
@philipwhiuk can I work on this issue?

**Comment by philipwhiuk** [comment]
Yep feel free - I don't think anyone else has looked into why it happens :)

**Comment by mhf1998** [comment]
This bug still exists in the latest version, 6.707. @cketti 

which means that when writing a message in the "Reply to", "To", "CC", and "BCC" fields with a non-email format, and then Rotate screen, the text is cleared after Rotate

Even if we write an email-formatted text in these fields, if the focus does not change to make it appear like a recipient, and then rotate the screen, we still see that the text is cleared.

**Comment by rezazarchi** [comment]
I think it should be fixed in the [library](https://github.com/splitwise/TokenAutoComplete). 
They handled state changes for most things, but this issue didn't handle.
